src/h5web/visualizations/heatmap/AxisGrid.tsx

```diff
@@ -54,16 +55,17 @@ function AxisGrid(props: Props): JSX.Element {
   });
 
   return (
-    <Dom>
-      <div
-        className={styles.axisGrid}
-        style={{
-          width,
-          height,
-          gridTemplateColumns: `${axisOffsets[0]}px 1fr`,
-          gridTemplateRows: `1fr ${axisOffsets[1]}px`,
-        }}
-      >
+    <Dom
+      className={styles.axisGrid}
+      style={{
+        width: width + leftAxisWidth,
+        height: height + bottomAxisHeight,
+        left: -leftAxisWidth,
```

Comment:
The axis grid is now inside the canvas, which no longer includes the size of the axis, so we need to increase the grid's size and offset it to give the axis some space and position them properly.

Reply:
I've added a comment here.